src/utils/credential.js

```diff
@@ -0,0 +1,72 @@
+import * as jwt from './jwt';
+import { decode } from 'jsonwebtoken';
```

Comment:
Hmm. I find it hard to justify this refactoring since it requires jsonwebtoken to be added on the browser side. Browser side scope should always be appUser anyway, so this credential thing has little to offer.

I'll keep some of these changes, but I think I'm going to get rid of this credential thing. I'll find another way.